roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -15,7 +15,7 @@ auth service required = none
 auth client required = none
 auth supported = none
 {% endif %}
-fsid = {{ fsid }}
+{% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %}
```

Comment:
I do not use this role just its templatate , already ceph-mon dockerized deployment uses it with or without kv backend https://github.com/ceph/ceph-ansible/blob/docker/roles/ceph-mon/tasks/docker/create_configs.yml#L12